src/pages/image/_id/report.vue

```diff
@@ -0,0 +1,99 @@
+<template>
```

Comment:
If we decide to make the top part of the single result and report layouts similar, would it be possible to extract the BackLink and the Image display to a separate component and reuse them in both pages?